_data/banking.yml

```
@@ -721,13 +901,17 @@ websites:
     facebook: ICBCArgentina
     img: icbc.png
     lang: es
+    regions:
+      - ar
 
   - name: ICICI
     url: https://www.icicibank.com
     img: icici.png
     tfa:
       - sms
     doc: https://www.icicibank.com/online-safe-banking/otp.page
+    regions:
+      - us
```

```suggestion
      - in
```
ICICI is a bank in India.